install-vdvs.ps1

```diff
@@ -0,0 +1,97 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Installation Instructions
+# ============================================================================
+# Run the script with the plugin url as the parameter in a PowerShell session:
+# PS C:\Users\Administrator> .\install-vdvs.ps1 <vdvs-download-url>
```

Comment:
PowerShell does not follow this convention. It uses Get-Help cmdlet to fetch help information for all cmdlets. We need to investigate how to interact with Get-Help. That might be trivial, but I think it can be addressed separately.